sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/DocumentSentiment.java

```diff
@@ -8,43 +8,44 @@
 import java.util.List;
 
 /**
- * The DocumentSentiment model
+ * The {@link DocumentSentiment} model that contains sentiment label of a document, confidence score of the sentiment
+ * label, and a list of {@link SentenceSentiment}.
  */
 @Immutable
 public final class DocumentSentiment {
-    private final SentimentLabel sentiment;
+    private final DocumentSentimentLabel sentimentLabel;
```

Comment:
Is there a reason why we change the property/attribute name from `sentiment` to `sentimentLabel`?

Reply:
Not a strong reason why should change to `sentimentLabel`. Will change back to sentiment.